seed/php-sdk/imdb/clientName/src/Core/Client/RetryMiddleware.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Seed\Core\Client;
+
+use GuzzleHttp\Exception\ConnectException;
+use GuzzleHttp\Promise as P;
+use GuzzleHttp\Promise\PromiseInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Throwable;
+
+class RetryMiddleware
+{
+    private const RETRY_STATUS_CODES = [408, 429];
+
+    /**
+     * @var callable(RequestInterface, array): PromiseInterface
+     * @phpstan-ignore missingType.iterableValue
```

Comment:
Can we add an explicit type here to prevent the `phpstan-ignore` comment? 

Can we just say `private callable $nextHandler` here?

Reply:
`callable` isn't allowed as a type, only in typedocs. 
The reference RetryMiddleware.php didn't type the array options out.
I think the point of the options is that it can contain anything, but not sure how to tell phpstan that's okay.
Without this ignore the `composer analyze` fails which is in our seed scripts.